File: apps/android/app/src/main/java/com/friday/assistant/service/FridayRecognitionService.java
package com.friday.assistant.service;

import android.content.Intent;
import android.speech.RecognitionService;
import android.util.Log;

public class FridayRecognitionService extends RecognitionService {
    private static final String TAG = "FridayRecognition";

    @Override
    protected void onStartListening(Intent recognizerIntent, Callback listener) {
        Log.d(TAG, "onStartListening");
    }

    @Override
    protected void onStopListening(Callback listener) {
        Log.d(TAG, "onStopListening");
    }

    @Override
    protected void onCancel(Callback listener) {
        Log.d(TAG, "onCancel");
    }
}

File: apps/android/app/src/main/java/com/friday/assistant/service/FridayRecognitionService.kt
package com.friday.assistant.service

import android.content.Intent
import android.speech.RecognitionService
import android.util.Log

/**
 * System RecognitionService required by VoiceInteractionService.
 */
class FridayRecognitionService : RecognitionService() {

    companion object {
        private const val TAG = "FridayRecogService"
    }

    override fun onStartListening(recognizerIntent: Intent?, listener: Callback?) {
        Log.d(TAG, "onStartListening")
    }

    override fun onStopListening(listener: Callback?) {
        Log.d(TAG, "onStopListening")
    }

    override fun onCancel(listener: Callback?) {
        Log.d(TAG, "onCancel")
    }
}
